exercises/concept/log-line-parser/.docs/instructions.md

```diff
@@ -0,0 +1,48 @@
+# Instructions
+
+In this exercise you'll be processing log-lines.
+
+Each log line is a string formatted as follows: `"[<LEVEL>]: <MESSAGE>"`.
+
+There are three different log levels:
+
+- `INFO`
+- `WARNING`
+- `ERROR`
+
+You have three tasks, each of which will take a log line and ask you to do something with it.
+There are lots of ways to solve these tasks - choose your favourite methods from the examples above and see what you can come up with.
+
+## 1. Get message from a log line
+
+Implement the `LogLineParser#message` method to return a log line's message:
+
+```ruby
```

Comment:
Do you want to mention you're showing Ruby code prior to showing Ruby-formatted code?

Reply:
I copied the exercise over from the ruby track: not quite finished with it yet. Thanks for the sharp eyed review!